llvm/test/Analysis/LoopAccessAnalysis/is-safe-dep-distance-with-loop-guards.ll

```diff
@@ -37,49 +37,3 @@ loop:
 exit:
   ret void
 }
-
-define void @safe_deps_2_due_to_dependence_distance(i16 %n, ptr %p3, i16 noundef %q, ptr %p1, ptr %p2) {
```

Comment:
Where has this moved to?

Reply:
This is now `safe_with_rtchecks_loopguards`.